config/webhook/kustomizeconfig.yaml

```diff
@@ -0,0 +1,25 @@
+# the following config is for teaching kustomize where to look at when substituting vars.
+# It requires kustomize v2.1.0 or newer to work properly.
+nameReference:
+- kind: Service
+  version: v1
+  fieldSpecs:
+  - kind: MutatingWebhookConfiguration
```

Comment:
Can we omit the MutatingWebhookConfiguration? we don't handle it.

Reply:
Correct. We can. I was waiting for your (all reviewers) feedback. Will remove.